README.md

```diff
@@ -2256,8 +2255,8 @@ A Guide like this can’t give you personalized advice on what a reasonable offe
 that depends greatly on your skills, the marketplace of candidates, what other offers you
 have, what the company can pay, what other candidates the company has found, and the
 company’s needs.
-However, we will cover the basics of what to expect with offers, and how to approach
-negotiations.
+But we can cover the basics of what to expect with offers, and provide some expert advice
```

Comment:
Nit: is "expert" necessary? I usually get suspicious if someone tells me they're giving expert advice.